### PR TITLE
create config set/unset without restart. closes #908

### LINKED ISSUE #908: dokku config:set but without restart (in a pre-release hook)
Hi,
I need to be able to call dokku config:set but without restarting the application, because I'm already in the process of restarting the application. I'm in a pre-release hook, see:
https://github.com/vincentfretin/dokku-elasticsearch-plugin/blob/master/pre-release#L37
How can we modify the config:set code to handle this case?
For now the application restarts twice if the elasticsearch container is not started.

### PR DESCRIPTION
**Pull request by michaelshobbs**
creates `config:set-norestart` and `config:unset-norestart`

I did not include these in the help output deliberately as this functionality is not compliant with Heroku functionality and is intended for internal use by things like plugins.

refs #1009

### CODE REVIEW

**Comment by michaelshobbs** [comment]
ping @josegonzalez

File: plugins/config/commands [hunk 1]
```diff
@@ -149,12 +165,29 @@ ${var}"
...
+    dokku ps:restart $APP
+  ;;
+
+  config:unset-norestart)
+    [[ -z $2 ]] && echo "Please specify an app to run the command on" && exit 1
+    verify_app_name "$2"
+    APP="$2"
+
+    if [[ -z $3 ]]; then
+      echo "Usage: dokku config:unset APP KEY1 [KEY2 ...]"
```
Comment (josegonzalez) [inline plugins/config/commands]: wrong usage

File: plugins/config/commands [hunk 2]
```diff
@@ -101,6 +100,23 @@ case "$1" in
...
+    dokku ps:restart $APP
+   ;;
+
+  config:set-norestart)
+    [[ -z $2 ]] && echo "Please specify an app to run the command on" && exit 1
+    verify_app_name "$2"
+    APP="$2"
+
+    if [[ -z "${*:3}" ]]; then
+      echo "Usage: dokku config:set APP KEY1=VALUE1 [KEY2=VALUE2 ...]"
```
Comment (josegonzalez) [inline plugins/config/commands]: wrong usage

**Comment by michaelshobbs** [comment]
@josegonzalez fixed usage message

**Comment by josegonzalez** [comment]
I actually thought about it more, and since it's being called by the external command, we should just use the external names (set/unset) as opposed to the actual. Can you revert that change? As well, add some docs to the `plugin development` section regarding it's usage.

**Comment by josegonzalez** [comment]
Once ^ is done, we can merge :)

**Comment by michaelshobbs** [comment]
Done and done.

**Pull request merged**